src/Rules.Framework.WebUI/Utitlies/PolymorphicWriteOnlyJsonConverter.cs

```diff
@@ -0,0 +1,19 @@
+namespace Rules.Framework.WebUI.Utitlies
+{
+    using System;
+    using System.Text.Json;
+    using System.Text.Json.Serialization;
+
+    internal sealed class PolymorphicWriteOnlyJsonConverter<T> : JsonConverter<T>
+    {
+        public override T Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            throw new NotImplementedException($"Deserializing not supported. Type={typeToConvert}.");
```

Comment:
We don't want to support this only for now, right?

Reply:
Correct

Reply:
Done

Reply:
Thank you! 👌